packages/tradershub/src/features/cfd/components/TradingAppCardLoader/TradingAppCardLoader.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const TradingAppCardLoader = () => {
```

Comment:
Guess we can just use arrow function without a block body since this does not contain logic and just returns JSX